scripts/apps/search/services/SearchService.ts

```diff
@@ -86,6 +86,23 @@ export function generateTrackByIdentifier(
     return getTrackByIdentifier(item._id, item.state !== ITEM_STATE.INGESTED ? item._current_version : null);
 }
 
+/**
+ * Determine if refresh button needs to be shown.
+ * It is shown if items are added/removed or order has changed.
+ */
+export function showRefresh(currentItems: Array<IArticle>, newItems: Array<IArticle>) {
+    for (let i = 0; i < newItems.length; i++) {
```

Comment:
would be good to check lengths as well, otherwise if last item is removed it won't catch it